src/Components/Blazor/Build/test/BindRazorIntegrationTest.cs

```diff
@@ -201,7 +201,7 @@ public static class BindAttributes
                 frames,
                 frame => AssertFrame.Element(frame, "div", 3, 0),
                 frame => AssertFrame.Attribute(frame, "myvalue", "hi", 1),
-                frame => AssertFrame.Attribute(frame, "myevent", typeof(Action<UIEventArgs>), 2));
+                frame => AssertFrame.Attribute(frame, "myevent", typeof(EventCallback), 2));
```

Comment:
@SteveSandersonMS - the types that get placed here depend on whether the delegate has a relevant `.Target` or not and whether bind is applied to a component or element. We already discussed this in detail, but I wanted to point it out for reviewing purposes.